arm-network/2017-06-01/examples/SubnetPrivateAccessServiceCreate.json

```diff
@@ -0,0 +1,49 @@
+{
+  "parameters": {

```

Comment:
similar comment as in the other example, missing required parameters, "subnetName", "subnetParamaters" with its properties "name", "addressPrefix" and "privateAccessServices" inside.